spec/controllers/email_addresses_controller_spec.rb

```diff
@@ -81,7 +81,7 @@
   describe "PUT #update" do
     context "with valid params" do
       let(:new_attributes) {
-        { address: 'belieber@example.com', person_id: person.id }
+        { address: 'belieber@example.com', contact_id: person.id, contact_type: 'Person' }
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.